cmd/kubelet/app/options/options.go

```diff
@@ -235,6 +235,7 @@ func (c *kubeletConfiguration) addFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&c.NetworkPluginDir, "network-plugin-dir", c.NetworkPluginDir, "<Warning: Alpha feature> The full path of the directory in which to search for network plugins or CNI config")
 	fs.StringVar(&c.CNIConfDir, "cni-conf-dir", c.CNIConfDir, "<Warning: Alpha feature> The full path of the directory in which to search for CNI config files. Default: /etc/cni/net.d")
 	fs.StringVar(&c.CNIBinDir, "cni-bin-dir", c.CNIBinDir, "<Warning: Alpha feature> The full path of the directory in which to search for CNI plugin binaries. Default: /opt/cni/bin")
+	fs.StringVar(&c.CNIVendorDirPrefix, "cni-vendor-dir-prefix", c.CNIVendorDirPrefix, "<Warning: Alpha feature> The full path of parent directory that houses all vendor CNI plugin binaries according to VendorCNIDirTemplate(\"<this-prefix>/opt/<cni-plugin-type>/bin\").  Default: \"\"")
```

Comment:
I am not sure what the original motivation to use `%s/opt/%s/bin` as VendorCNIDirTemplate (https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/network/cni/cni.go#L39)
I think conventionally when a host directory is mounted into a container a similar template is followed. (ex. `/opt/calico/bin` from the host is mounted into install-calico container as `/host/opt/calico/bin`)